src/main/java/com/fontys/api/service/TeamService.java

```diff
@@ -12,32 +13,42 @@
 @Service
 public class TeamService {
 
-    private final
-    TeamRepository teamRepository;
+    private final TeamRepository teamRepository;
+    private final UserRepository userRepository;
 
-    public TeamService(TeamRepository teamRepository) {
+    public TeamService(TeamRepository teamRepository, UserRepository userRepository) {
         this.teamRepository = teamRepository;
+        this.userRepository = userRepository;
     }
 
     @Transactional
     public Team createTeam(String name) {
         return teamRepository.save(new Team(name));
     }
 
-    @Transactional
-    public void deleteTeam(Long id) {
-        Optional<Team> team = teamRepository.findById(id);
-        team.ifPresent(teamRepository::delete);
-
-    }
-
     @Transactional(readOnly = true)
     public List<Team> getAllTeams() {
         return teamRepository.findAll();
     }
 
     @Transactional(readOnly = true)
-    public Optional<Team> getTeam(Long id) {
+    public Optional<Team> getTeam(Integer id) {
         return teamRepository.findById(id);
     }
+
+    @Transactional
+    public String deleteTeam(Integer id) {
+        Optional<Team> team = teamRepository.findById(id);
+        team.ifPresent(teamRepository::delete);
+        return "Team " + team.get().getName() + " deleted";
+    }
+
+    @Transactional
+    public String addUserToTeam(Integer teamId, Integer userId) {
```

Comment:
Manual testing shows that this function doesn't actually work

Reply:
It only works if a user and team are created, as should always be the case

Reply:
I tested the function in the right order, but if I did it wrong, then I should have gotten a usefull exception